.gitignore

```diff
@@ -160,3 +160,17 @@ pip-selfcheck.json
 # End of https://www.gitignore.io/api/osx,python
 
 doc/api/
+
+
+build-lambda.sh
+function.zip
+python-orchestrator.zip
+python-worker.zip
+test-out.json
+test.json
+test13.json
+tests/aws_lambda/worker/commands.sh
+tests/aws_lambda/worker/function.zip
+tests/aws_lambda/worker/python-worker.zip
+tests/aws_lambda/worker/test.json
+tmp.py
```

Comment:
Looks like a bunch of these aren't used anymore so most of it could be reset.

Reply:
Fixed in 81d4c53dbbfbe047a2152da012c6607435c32954